examples/mobilegeolocation.html

```diff
@@ -12,6 +12,7 @@
       <div id="map" ngeo-map="::ctrl.map"></div>
       <button ngeo-geolocation=""
         ngeo-geolocation-map="::ctrl.map"
+        ngeo-geolocation-mainctrl="::ctrl"
```

Comment:
For me, you don't need the full main controller =>
```suggestion
        ngeo-geolocation-loading="::ctrl.loading"
```

Reply:
I agree, it would make more sense.
But I tried to do so (in GMF 2.4 or maybe 2.5) and the ``loading`` property value was passed once for all and never updated.
Could it be different in master? Or perhaps the attribute syntax is not correct? I had tried removing the one-time binding ``::`` but with no more success.

By the way would you have a suggestion where this functionality should be demonstrated? The current ``mobilegeolocation`` has a very simple main controller with no ``loading`` attribute...

Reply:
I just remove the one-time binding from the suggestion :-)
@fredj @ger-benjamin any input?

Reply:
it will not work as it because the scope is not bound to the controller.
- add `bindToController: true` in `geolocationComponent`
- add `$onInit` function to the controller and move the `getMapFn` function call
- handle `loading` value change in `$onInit`

Reply:
Then @fredj what do you preconize to do?
Keep like it or bindToController, ...?